networking/multiple-networks/configuring-bridge.adoc

```diff
@@ -0,0 +1,22 @@
+[id="configuring-bridge"]
+= Configuring a bridge network
+include::modules/common-attributes.adoc[]
+:context: configuring-bridge
+
+toc::[]
+
+As a cluster administrator, you can configure an additional network for your
+cluster using the bridge Container Network Interface (CNI) plug-in. When
+configured, all Pods on a node are connected to a virtual switch. Each Pod is
+assigned an IP address on the additional network.
+
+// TODO - include specifics of virtual bridge
+
+include::modules/nw-multus-create-network.adoc[leveloffset=+1]
+include::modules/nw-multus-bridge-object.adoc[leveloffset=+2]
+include::modules/nw-multus-ipam-object.adoc[leveloffset=+2]
+include::modules/nw-multus-add-pod.adoc[leveloffset=+1]
+include::modules/nw-multus-remove-pod.adoc[leveloffset=+1]
+include::modules/nw-multus-edit-network.adoc[leveloffset=+1]
+include::modules/nw-multus-delete-network.adoc[leveloffset=+1]
```

Comment:
Deleting a network isn't part of configuring it. This, too, should be in a different assembly.